src/components/Nav/BottomButtons.tsx

```diff
@@ -0,0 +1,19 @@
+import {Button} from "../Common/Button"
+import { AiOutlineGithub } from "react-icons/ai";
+import { WithTooltip } from "../Tooltip"
+import { SiGithub } from "react-icons/si";
+
+function navigateToGithub(){
+window.open("https://github.com/MelkeyOSS/astrostation")
```

Comment:
Hey here is a suggestion, using a `<a>` tag is going to be much better as a semantic HTML element.

So you would need to remove the `navigateToGithub()` function and make the button below use an anchor (`<a>`) tag.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a